code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -923,37 +889,6 @@ nobiliumsuppression = INFINITY
 			air.temperature = max(((temperature * old_heat_capacity - energy_used) / new_heat_capacity), TCMB)
 	return REACTING
 
-/datum/gas_reaction/hexane_plasma_filtering
-	priority = 8
```

Comment:
You can't just change the comment, you need to change all these priority variables of everything above it.

Reply:
fuck